spec/shared/context/service_record_declaration_params.rb

```diff
@@ -14,6 +14,9 @@
   let(:ect_params) do
     params.merge({ cpd_lead_provider: cpd_lead_provider })
   end
+  let(:ect_params_with_different_date) do
+    ect_params.merge({ declaration_date: (ect_declaration_date + 1.second).rfc3339 })
+  end
```

Comment:
Does this really belong in shared context? We seem to be using it just once.

Reply:
Moved to the test in 9a99d84